src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java

```diff
@@ -166,17 +196,36 @@ public void userClicksAddNewCondition() {
 	@And("a user enters patient condition")
 	public void enterExistingCondition() {
 		conditionPage.typeInCondition(CONDITION);
+		conditionPage.clickSave();
+	}
+
+	@And("the user enters patient condition Acute malnutrition")
+	public void enterOneOfExistingConditions() {
+		conditionsPage.clickOnAddNewCondition().waitForPage();
+		conditionPage.typeInCondition("Acute malnutrition");
 	}
 
 	@And("a user clicks on save condition button")
 	public void saveCondition() {
 		conditionPage.clickSave();
+		conditionPage.waitForPage();
```

Comment:
Remove this condition for the driver to easily save without waiting for the page.